frontend/src/routes/Home/Overview/OverviewPage.tsx

```diff
@@ -292,7 +292,8 @@ export default function OverviewPage() {
             const policyReportCriticalCount = policyReportsForSelectedClusters.reduce(
                 (total: number, currentValue: PolicyReport) => {
                     const criticalResults = currentValue.results.filter(
-                        (result: PolicyReportResults) => result.properties.total_risk === '4'
+                        (result: PolicyReportResults) =>
+                            result.source === 'insights' && result.properties.total_risk === '4'
                     )
                     return total + criticalResults.length
```

Comment:
I would suggest adding a function to create this reducer function, with an argument for the total_risk value, rather than repeating it 4 times.

Reply:
Good idea - I changed it a little bit to reduce the number of loops we were previously running.